### PR TITLE
Add Notifications and Skinned products 

### PR DESCRIPTION
**Pull request by BigSmoKe07**
Added a Advance notification which shows the picture of animal skinned 
Added around  150 SkinnableAnimals and Items according to each one

### CODE REVIEW

File: config.lua [hunk 1]
```diff
@@ -13,25 +13,26 @@ Config.Language = {
...
 Config.keys = {
     G = 0x760A9C6F, -- butcher sell and Stow
 }
 
+Config.aiButcherped = true -- spawn ai butched ped set to false if you dont want an ai butcher ped to spawn
+
 Config.joblocked = false -- lock the butcher so only people with the job can access. u can change access to each butcher by editing this  butcherjob = "butcher"
 
-Config.maxpelts = 2 -- max pelts allowed on back of horse (Max 3 allowed)
+Config.maxpelts = 5 -- max pelts allowed on back of horse
```
Comment (AndrewR3K) [inline config.lua]: Everything looks great, but can you please put this back to what its supposed to be. The limit is 3 for technical limitations.

```lua
Config.maxpelts = 2 -- max pelts allowed on back of horse (Max 3 allowed)
```

Comment (BigSmoKe07) [reply config.lua]: Sure, Limit changed to 3

**Comment by outsider31000** [comment]
image
use the API please you can find it in discord the documentation.

image
why have this been removed?

File: config.lua (hunk 1 above)
Comment (BigSmoKe07) [reply config.lua]: added the butcherped values

Comment (BigSmoKe07) [reply config.lua]: changed notification to new API

**Pull request merged**